utils/core-utils/src/utils/solidity/compiler.ts

```diff
@@ -88,11 +88,7 @@ async function _getCompiler(version: string): Promise<any> {
   const isDownloaded = await downloader.isCompilerDownloaded(version);
 
   if (!isDownloaded) {
-    await downloader.downloadCompiler(
-      version,
-      async () => {},
-      async () => {}
-    );
+    await downloader.downloadCompiler(version);
```

Comment:
Looks like this is causing CI to fail: https://app.circleci.com/pipelines/github/Synthetixio/synthetix-v3/3296/workflows/ce27a610-0b90-48ad-b23f-47e44855398e/jobs/56098

Reply:
Oh yeah I remember fixing that somewhere. hold on

Reply:
haha indeed
![image](https://github.com/user-attachments/assets/6fbf901d-1243-48f3-860f-4a98b0340c80)